src/.env.sample

```
@@ -39,4 +39,8 @@ VITE_API_BASE_URL=http://localhost:8000
 
 # E2E tests login user
 CORPORATE_EMAIL=your-email
-PASSWORD=your-password 
\ No newline at end of file
+PASSWORD=your-password 
+
+# Redis
+REDIS_HOST=localhost
+REDIS_PORT=6379
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)